app/scripts/utils/dict-items.js

```diff
@@ -6,7 +6,7 @@ const dictItems = (dictionary) => {
   const keyValues = [];
 
   for (const key in dictionary) {
-    if (dictionary.hasOwnProperty(key)) {
+    if (dictionary.hasOwnProperty(key)) {  // eslint-disable-line no-prototype-builtins
```

Comment:
They are not equivalent:
```
({}).hasOwnProperty('toString')
false
'toString' in {}
true
```
... That said, it's always possible that `in` is the behavior we actually want, but it would be a change in behavior, and I didn't want to get into that.